js/src/vue/app.js

```diff
@@ -69,9 +69,12 @@ const component_context = import.meta.webpackContext('.', {
     chunkName: '/vue-sfc/[request]'
 });
 
-/* global __webpack_public_path__ */
-// eslint-disable-next-line no-global-assign
-__webpack_public_path__ = CFG_GLPI.root_doc + __webpack_public_path__;
+// TODO: CFG_GLPI is undefined sometimes, find out why.
+if (typeof CFG_GLPI !== "undefined") {
```

Comment:
This is not a good idea. We have to find why E2E test fails here.

@cconard96 Do you have an idea ?

Reply:
Yes but it is not related to the fix from this PR, it should be looked at independently.

Reply:
Fixed in https://github.com/glpi-project/glpi/pull/19721